### PR TITLE
User can view a list of movies - issue #1

### PR DESCRIPTION
**Pull request by yuliia-p**
Screen Shot 2022-08-02 at 10 59 31 AM
Screen Shot 2022-08-02 at 10 59 19 AM

### CODE REVIEW

**Comment by yuliia-p** [comment]
https://user-images.githubusercontent.com/106217355/182444041-39dae0eb-845b-471d-b270-df5129bc7254.mp4

File: css/layout.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
+.container {
+  /* max-width: 575px; */
+
+  /* margin: 0 auto; */
```
Comment (codingcodymiller) [inline css/layout.css]: Is there any reason this is commented out?

File: css/styles.css [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+.content-holder {
+  background-color: #1e1e29;
+  border-radius: 10px;
+}
+
+.margin {
+  margin: 11px 11px 0;
+}
+
+.margin-rigth {
```
Comment (codingcodymiller) [inline css/styles.css]: `rigth` should be `right`

File: js/data.js [hunk 3]
```diff
@@ -1 +1,15 @@
...
+//   xhr.addEventListener('load', function () {
+//     console.log('xhr.response: ', xhr.response);
+//     console.log('xhr.response.items: ', xhr.response.items);
+//     console.log('xhr.response.items[0]: ', xhr.response.items[0]);
+//     console.log('xhr.response.items[0].crew: ', xhr.response.items[0].crew);
+//   });
+//   xhr.send();
+//   // return xhr.response.items;
+// }
+// getMovies();
```
Comment (codingcodymiller) [inline js/data.js]: Please remove commented code prior to submitting for review

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css/styles.css (hunk 2 above)
Comment (yuliia-p) [reply css/styles.css]: Screen Shot 2022-08-02 at 11 38 10 AM

File: js/data.js (hunk 3 above)
Comment (yuliia-p) [reply js/data.js]: Screen Shot 2022-08-02 at 11 41 19 AM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**